tarsplitter.go

```diff
@@ -54,17 +57,31 @@ func doSplit() {
 		os.Exit(1)
 	}
 
-	file, err := os.Open(*input)
-	fatalIf(err, "Failed statting input", *input)
-	defer file.Close()
+	var file io.Reader
+	var partSizeBytes int64
```

Comment:
I would use `uint64` here since sizes are always positive, but didn't want to refactor the other `int64` usages (`bytesBeforeWrite`, etc.).